Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 
 gem 'mongo_mapper', '0.12.0'
+gem 'mongo', '1.11.1'
+gem 'plucky', '0.5.2'
```

Comment:
plucky is a requirement of mongo_mapper.  I don't think you want to lock it, do you?